### DISCUSSION TITLE
How can I make `@tailwindcss/jit` work?

### DISCUSSION THREAD

**Discussion by vojtaholik** [jaredpalmer/razzle#1585] [Q&A] [answered]
How can I make tailwind's new [just-in-time compiler](https://www.npmjs.com/package/@tailwindcss/jit) work with Razzle?

Following doesn't seem to work:

```js
// postcss.config.js

module.exports = {
  plugins: {
    // tailwindcss: {}, // this works
    '@tailwindcss/jit': {}, // this does not
    autoprefixer: {},
    'postcss-nested': {},
  },
}
```

**Comment by fivethreeo** [comment]
I suspect this only works with style loader and not with mini-css-extract loader/plugin or SSR.

**Comment by fivethreeo** [comment, marked as answer]
So use only spa mode and modify css rule to use only style loader and remove mini-css-extract plugin
